### ISSUE TITLE
Button

### ISSUE THREAD

**Issue by PabloAlice** [windingtree/wt-ui-react#8]
**Is your feature request related to a problem? Please describe.**
we don't have Buttons components 
**Describe the solution you'd like**
I would like to have a Button component that keeps mostly the native button API and adds some props to make it nicer and easier to use.

**Describe alternatives you've considered**
it can follow [this API as example](https://react-bootstrap.github.io/components/buttons/) trying to avoid getting coupled to Bs.
There are more libraries out there that can be checked to take some improvements.

**Additional context**
the buttons should be tighted to wt-ui styles
we should spread additional props to our buttons to allow users access to native button props.

**Comment by MatiasOS** [comment]
Interface for the component 
```javascript
type Props = {
  block?: bool,
  children?: React$Node,
  disabled?: bool,
  href?: string,
  kind?: 'primary' | 'secondary' | 'light' | 'dark' | 'link',
  onClick: string => void,
  outlined?: bool,
};
```

With default props 
```javascript
Button.defaultProps = {
    kind: 'primary',
};
```

@PabloAlice @skkr  let me know what you think about this approach.

**Comment by PabloAlice** [comment]
@MatiasOS Great Job!
some thoughts that come to my mind:
 - children shouldn't be optional (you can't show a button with anything inside right?)
 - we should expose a className prop and merge it with the current classes (we can use "classnames", lt's an npm package)
 - Absolutely agree with having an outlined prop

**Comment by skkr** [comment]
Looks good! 

For your consideration :

Well probably have two styles. Primary solid green and secondary (outlined white and black). So is possible it won't match bootstrap class names and styles exactly. 

Buttons can have children, icons basically. Still under review, but at least I can think in the loader. Could be cool to have a prop, loading to enable and disable that state.

There are other button variants we might include, like buttons with dropdowns, but is not defined yet.

**Comment by MatiasOS** [comment]
@PabloAlice  I will add `className` as prop.

@skkr Do you mean than only we have 4 button styles?  primary, secondary   and both outlined?
 What changes `loading` prop from `disabled`?

**Comment by skkr** [comment]
Probably just two styles, primary and secondary. Secondary can be dark or light depending over what background is used.

**Comment by MatiasOS** [comment]
closed by #11
